server/src/main/java/org/apache/iotdb/db/mpp/sql/planner/plan/node/write/InsertRowsNode.java

```diff
@@ -100,6 +104,16 @@ public List<PlanNode> getChildren() {
   @Override
   public void addChild(PlanNode child) {}
 
+  @Override
```

Comment:
According to MPP design, planNode is only for describe the plan infomation, should not contain operation function. You can use visitor mode to add some operation in the PlanVisitor.